internal/exec/helmfile.go

```diff
@@ -275,11 +278,6 @@ func checkHelmfileConfig(cliConfig schema.CliConfiguration) error {
 			return errors.New("Helm AWS profile pattern must be provided in 'components.helmfile.helm_aws_profile_pattern' config or " +
 				"'ATMOS_COMPONENTS_HELMFILE_HELM_AWS_PROFILE_PATTERN' ENV variable")
 		}
-
-		if len(cliConfig.Components.Helmfile.ClusterNamePattern) < 1 {
-			return errors.New("Cluster name pattern must be provided in 'components.helmfile.cluster_name_pattern' config or " +
```

Comment:
@aknysh do we still require this if use_eks is true?

Reply:
@shahincsejnu we should require `cluster_name_pattern` if `use_eks` is true

Reply:
@osterman okay, made it required!